Login/src/main/java/com/bms/login/payload/response/UserInfoResponse.java

```diff
@@ -0,0 +1,49 @@
+package com.bms.login.payload.response;
+
+import java.util.List;
+
+public class UserInfoResponse {
```

Comment:
Can use @Builder & @Value annotations to make the response immutable

Reply:
okay